src/App.js

```diff
@@ -9,7 +10,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article
+    title={article.title}
+    text={article.text}
+    date={article.date}
+  />
```

Comment:
Better way to pass object properties as props:
```suggestion
  <Article. {...article} />
```